### DISCUSSION TITLE
Development Hardware - Help Wanted!

### DISCUSSION THREAD

**Discussion by gtxaspec** [themactep/thingino-firmware#194] [General]
Hey there! We're seeking individuals with the specific devices listed below to help expand support for Thingino. If you have Linux experience and are eager to dive into embedded systems, we'd love to hear from you to collaborate with our developers to get these devices running with your help! You can also contribute by donating hardware to our developers, which will accelerate the development and support process.

- Atom
  - AtomCam (Original) 
  - AtomCam Swing
 - Tapo
   - C200 
- Wansview
  - W9 
 - Wyze
   - Floodlight
   - Garage Door Controller 
   - Sense Bridge USB Dongle

**Comment by ikout** [comment]
I have a Tapo C200 that I'm OK to mess with, the Pan-Tilt has issues so I not using that anymore and I don't mind to play with.
I'm getting a new C100 to test this firmware and if it works as I hope that will be great if I can switch to this one.

**Comment by 5310H** [comment]
I have one of my floodlights on the bench so I'd be happy to help.  Been waiting til someone figure outs how to just turn the light on and off.  Everything else other than the PIR,  which never worked, can be done within Thingino's motion and dusk/dawn features.  Have motion, run floodlight script to turn light on if dark, sleep so many seconds then turn light off and wait a set cool down period before another activation.

**Comment by ryang3d** [comment]
I have a bunch of Wyze Floodlights (V1) installed and one still in the box. Would love to help find a way to gain control of the lights of the actual floodlight as well as the PIR sensor (I just want to disable it! - might just paint over it or something for now xD)

**Comment by pursual** [comment]
I see images for the Floodlight on the main page, but cant find any documentation on controlling the floodlight.

**Comment by MinnesotaMean** [comment]
> Hey there! We're seeking individuals with the specific devices listed below to help expand support for Thingino. If you have Linux experience and are eager to dive into embedded systems, we'd love to hear from you to collaborate with our developers to get these devices running with your help! You can also contribute by donating hardware to our developers, which will accelerate the development and support process.
> 
> * Atom
>   
>   * AtomCam (Original)
>   * AtomCam Swing
> * Tapo
>   
>   * C200
> * Wansview
>   
>   * W9
> * Wyze
>   
>   * Floodlight
>   * Garage Door Controller
>   * Sense Bridge USB Dongle

I have have two garage door opener controllers and would absolutely love to get these intergrated to my thingino wyze cams! I am not super experienced but feel like I can learn and follow any instructions.

**Comment by nkm8** [comment]
I am also very interested in Wyze floodlight support. I currently have 2 devices available - 1 still running stock Wyze and 1 running thingino.

**Comment by Myrcurial** [comment]
Tag another case of "I've got a floodlight, would like to control it" -- less interested in the PIR than just turning the big LEDs on and off. I've got one that is in a malfunctioning state that I'd be happy to open up and do some photography if that would help too. The other one is on the side of the house with a correctly flashed thingino V3 connected.

**Comment by jimoquinn** [comment]
Is this still a valid list?  I'd be happy to send a Tapo C200 to one of the developers.  How can we arrange for this to be done?

**Comment by ryang3d** [comment]
Just gonna throw this in here: I forked the [g8keeperzuul/wz_mini_hacks-MQTT](https://github.com/g8keeperzuul/wz_mini_hacks-MQTT) repo a little while ago and modified it a bit to get Wyze Floodlight control (or Spotlight control -separate branch-) via MQTT (and more specifically Home Assistant). Although it could still use some work ([see issues](https://github.com/ryang3d/wz_mini_hacks-MQTT/issues)), this is working quite well for me so far, and thought it might be useful here.

**Comment by ELY3M** [comment]
I have wyze floodlight right now.  I have it running on latest thingino version (2-23-26)    I use the bin for floodlight
only thing I have issue is not able to control the lights.   I am digging and trying to figure out how to control them.

**Comment by ELY3M** [comment]
Actually , I figured out!!!!   I had to convert the floodlight script to linux EOL   


3. Convert Windows Line Endings (CRLF) 
If you edited the script on a Windows computer, it likely has CRLF line endings, which Linux/Thingino does not understand. This often results in a "file not found" error because the system misinterprets the newline character. 

    Fix: Run dos2unix on your script:
    bash

    dos2unix yourscript.sh

    Alternatively, open the file in a text editor like Notepad++ or VS Code and change the EOL (End of Line) conversion to Unix (LF).

**Reply by bbfreak444** [reply]
I am trying to get the floodlight running now - any tips would be much appreciated. I have V3 firmware flashed, but the floodlight is always on. Are there any guides for this yet?
